src/components/Navbar/NavbarComponent.tsx

```diff
@@ -65,7 +67,18 @@ const NavbarComponent: FC<NavbarComponentProps> = ({
           {chainId && <NetworkButton chainId={chainId} />}
           <AccountButton {...{ openWalletModal, deactivate, account }} />
         </Stack>
-        <WalletConnectionAlert {...{ account, chainId }} />
+        <VStack
+          position="absolute"
+          w="fit-content"
+          paddingRight="40px"
+          top="94px"
+          right="5.25rem"
+          zIndex="10"
+          ml="4rem"
+        >
```

Comment:
Good idea moving the styles here to the `VStack` 👍 I would just slightly improve it, because it looks little off when two errors are displayed at the same time:

<img width="1052" alt="image" src="https://github.com/threshold-network/token-dashboard/assets/40306834/eb179f65-4c67-4863-b5bb-f089141a81bb">

and on smaller resolution:

<img width="1028" alt="image" src="https://github.com/threshold-network/token-dashboard/assets/40306834/525d83af-2df7-42a4-8e5d-8910b60aba1b">


Here, let's add `maxWidth` property to ensure it looks good on smaller screens. We should also align the items to the right because we are removing the `w="fit-content"` here which we will add it to each alert (so they will have different width.

```suggestion
        <VStack
          position="absolute"
          paddingRight="40px"
          top="94px"
          right="5.25rem"
          zIndex="10"
          ml="4rem"
          maxWidth={{ base: "38rem", xl: "100%" }}
          alignItems={"flex-end"}
        >
```

Reply:
Addressed in [apply styling to alert related components](https://github.com/threshold-network/token-dashboard/pull/736/commits/39e6e2d006412856ad90c9c80e4f9d712ea3d6e6)